sdk/daml-lf/snapshot/src/benchmark/scala/com/digitalasset/daml/lf/testing/snapshot/ReplayBenchmark.scala

```diff
@@ -32,6 +33,10 @@ class ReplayBenchmark {
   // path of the ledger entries - i.e. path to file of saved transaction trees
   var entriesFile: String = _
 
+  @Param(Array("V1", "V2"))
```

Comment:
I think it is a good idea to make the benchmark with both Id. 
However, I will keep only one as default.
Also you have to update the sdk/daml-lf/snapshot/README.md.